BotMain.py

```diff
@@ -15,14 +24,20 @@
     print(bot.user.name)
     print(bot.user.id)
     print('------')
-    await bot.change_presence(game=discord.Game(name="Being a useless bot"))
+
+    if 'dev_mode' in bot.config and bot.config['dev_mode']:
+        game = "[DevMode]"
+    else:
+        game = "Being a useless bot"
```

Comment:
The reason I didn't put it there is because I was unsure if we'd be using things like shard_count or any kind of variable accessible only through discord.py or Python in general.